plugins/modules/omevv_vcenter_info.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 9.8.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: omevv_vcenter_info
+short_description: Retrieve all or specific vCenter information.
+version_added: "9.8.0"
+description:
+  - This module retrieve all or specific vCenter information.
+extends_documentation_fragment:
+  - dellemc.openmanage.ome_auth_options
+options:
+  vcenter_hostname:
+    description:
+      - vCenter IP address or hostname.
+      - If I(vcenter_hostname) is provided, then module will retrieve only the specified vCenter details.
```

Comment:
- If I(vcenter_hostname) is provided, the module retrieves only specified vCenter information.